### PR TITLE
Add support for broker related commands

### PR DESCRIPTION
**Pull request by kirederik**
* create-service-broker
* delete-service-broker
* enable-service-access

with @deniseyu

### CODE REVIEW

File: assets/out [hunk 1]
```diff
@@ -229,6 +229,27 @@ echo "$params" | jq -c '.commands[]' | while read -r options; do
     printf '\e[92m[INFO]\e[0m Executing \e[33m%s\e[0m: %s\n' "$command" "$app_name"
     cf_target "$org" "$space"
     cf_bind_service "$app_name" "$service_instance" "$configuration"
+  elif [ "create-service-broker" = "$command" ]; then
+    broker_name=$(echo $options | jq -r '.broker_name //empty')
+    username=$(echo $options | jq -r '.username //empty')
+    password=$(echo $options | jq -r '.password //empty')
+    url=$(echo $options | jq -r '.url //empty')
+    is_space_scoped=$(echo $options | jq -r '.space_scoped //"false"')
+    printf '\e[92m[INFO]\e[0m Executing \e[33m%s\e[0m: %s %s %s %s %s\n' "$command" "$broker_name" "$username" "$password" "$url"
```
Comment (pcrocker-pivotal) [inline assets/out]: To be consistent with other log messages, this should only output the command and the broker name (and definitely not the password)

Comment (patrickcrocker) [reply assets/out]: Bah, i'll go ahead and change this :-)

**Comment by patrickcrocker** [comment]
This is now implemented in `pivotalpa/cf-cli-resource:2.3.0`

Please read the release notes as I have changed the parameter names to be consistent with the cf cli examples: https://github.com/Pivotal-Field-Engineering/cf-cli-resource/releases/tag/v2.3.0

**Comment by patrickcrocker** [comment]
Oh, I also implemented `disable-service-access`.

**Pull request merged**